txnkv/rangetask/range_task.go

```diff
@@ -190,9 +190,9 @@ Loop:
 
 		rangeEndKey, err := s.store.GetRegionCache().BatchLoadRegionsFromKey(bo, key, s.regionsPerTask)
 		if err != nil {
-			logutil.Logger(ctx).Info("range task failed",
+			logutil.Logger(ctx).Info("range task try to get range end key failure",
 				zap.String("name", s.name),
-				zap.String("startKey", kv.StrKey(startKey)),
+				zap.String("startKey", kv.StrKey(key)),
```

Comment:
I think we can retain the `startKey` and add a new filed eg `loadingRegionStartKey`

Reply:
fixed in [379918a](https://github.com/tikv/client-go/pull/657/commits/379918a5f3cff59412ad42800596eb656b2d0079)